pyrefinebio/config.py

```diff
@@ -5,10 +5,26 @@
 
 
 class Config:
+    """Config for pyrefinebio.
+
+    When instantiated, configurations are pulled first from environment variables,  
+    then from the config file locaed at `~/.refinebio.yaml`. If neither exist, default  
+    values are used.
+
+    The config instance should never be constructed or modified manually.  
```

Comment:
Is there any particular reason for this?

Reply:
Actually, our config is small enough that maybe we could even provide an example of the config here in case they do want to write it manually?

Reply:
idk, I just thought it would be better for the user to interact with the config solely through the config file or environment variables. Except when dealing with the token which is handled through the `Token` class.

Reply:
Oh whoops, I thought you were talking about the config file. I think manually modifying the instance might be fine as a way to temporarily change settings if someone figures it out, and I'm not sure what manually constructing an instance would look like.